pkg: cmd: support start/stop for OKD 4.1 provider

ffromani opened the pull request:
This patch enables starting/stopping OKD 4.1 clusters.

Signed-off-by: Francesco Romani <fromani@redhat.com>

ffromani committed:
pkg: cmd: support start/stop for OKD 4.1 provider

This patch enables starting/stopping OKD 4.1 clusters.

Signed-off-by: Francesco Romani <fromani@redhat.com>

fromanirh commented:
/cc @ksimon1 WIP

ffromani committed:
project: switch to go modules

Signed-off-by: Francesco Romani <fromani@redhat.com>

ffromani committed:
podman: find: more debug output

Also log the prefix we are looking for.

Signed-off-by: Francesco Romani <fromani@redhat.com>

ffromani committed:
show: add option to emit only container IDs

This option is going to be useful for the OKD cluster
support.

Signed-off-by: Francesco Romani <fromani@redhat.com>

fromanirh commented:
@ksimon1 I'll leave to you to handle the merge of this PR and the OKD support in general. I think this should be merged after the yet-to-be-posted PR to add the `copy-from` subcommand.

ffromani committed:
status: enable also okd-4.3

From kubevirtci provider's perspective, if okd-4.1 works,
then okd-4.3 should work too. Enable them both.

Signed-off-by: Francesco Romani <fromani@redhat.com>

fromanirh commented:
We also need this PR on kubevirtci side: https://github.com/kubevirt/kubevirtci/pull/183

ksimon1 reviewed (approved):
LGTM. This PR is needed for another functionality introduced in: https://github.com/fromanirh/pack8s/pull/27
